test-end-to-end/android-maestro/e2e-tests/001_004_admin_can_move_content.yaml

```diff
@@ -0,0 +1,35 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/001_004_admin_can_move_content"
+  TESTNAME: "001_004_admin_can_move_content"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+- runFlow:
+    file: "../common/add_content.yaml"
+    env:
+      TESTFILENAME: Epub_Content1.epub
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+- tapOn: "SAVE"
+# Add new folder to move content
+- tapOn:
+    id: "floating_action_button"
+- tapOn: "New folder"
+- tapOn: "Title*"
+- inputText: "Test Folder"
+- tapOn: "SAVE"
+- takeScreenshot: "${TESTRESULTSDIR}/screenshots/before_moving_content_to_new_folder"
+- longPressOn:
+    text: "E-book"
+- tapOn: "Move"
+- tapOn: "Folder"
+- tapOn: "Move entries to this folder"
```

Comment:
Before tapOn "Move entries to this folder" you need an assertion to check that you are now in the expected folder. 

I think you can use assertVisible on the title id - check that the title has changed to folder. Otherwise we risk the same thing as we had with cypress.